test/sql/test_text.py

```diff
@@ -673,6 +729,15 @@ def test_having(self):
     def test_from(self):
         self._test(select([table1.c.myid]).select_from, "mytable", "mytable")
 
+    def test_table_kw(self):
```

Comment:
**mike bayer** (zzzeek) wrote:

these are good tests.   However, let's move them into test/sql/test_compiler.py in the "class SchemaTest" class, as test_text() is mostly about using plain text in queries.  let's call them "test_schema_lowercase_XYZ" like "test_schema_lowercase_select", and we dont need to test the literal_column use case, just plain selects.   Also a test that includes select([table]).apply_labels() to test the extended label format.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/1944

Reply:
Do you want the `select` tests combined into one `test_schema_lowercase_select` - that seems to be common in `test_compiler.py`

Reply:
OK, it is common in test_compiler.py because I wrote those tests 15 years ago :)    one test per thing tested is best.     test_compiler.py ultimately should be rewritten in this style.

Reply:
ok - sounds good!